Python/src/xpc.py

```diff
@@ -1,20 +1,22 @@
 import socket
 import struct
 
+
 class XPlaneConnect(object):
-    '''XPlaneConnect (XPC) facilitates communication to and from the XPCPlugin.'''
+    """XPlaneConnect (XPC) facilitates communication to and from the XPCPlugin."""
+
     socket = None
 
     # Basic Functions
-    def __init__(self, xpHost = 'localhost', xpPort = 49009, port = 0, timeout = 100):
-        '''Sets up a new connection to an X-Plane Connect plugin running in X-Plane.
+    def __init__(self, xpHost="localhost", xpPort=49009, port=0, timeout=100):
```

Comment:
Is there any specific reason to change the single quotes to double? If not, I'd really prefer to just leave things as they are.

I really like the white space around my assignment operators as well.